src/main/java/frc/robot/commands/RaiseArmCommand.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.utils.MoPrefs;
+
+public class RaiseArmCommand extends CommandBase {
+    private final ArmSubsystem arms;
+    private final double power;
+    private final BooleanSupplier shouldBeParallel;
+    private boolean wasParallel;
+
+    public RaiseArmCommand(ArmSubsystem arms, double power, BooleanSupplier parallel) {
+        this.arms = arms;
+        this.power = power;
+        this.shouldBeParallel = parallel;
+    }
+
+    @Override
+    public void execute() {
+        this.arms.adjustArm(power);
+
+        boolean isParallel = this.shouldBeParallel.getAsBoolean();
+        if (this.wasParallel && !isParallel) {
+            this.arms.idleWrist();
+        } else if (isParallel) {
+            this.arms.adjustWrist(power * MoPrefs.armJointSetpointRatio.get());
+        }
+
+        this.wasParallel = isParallel;
+    }
+
+    @Override
+    public boolean isFinished() {
+        return false;
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        this.arms.idleArm();
+
+        if (this.wasParallel) {
+            this.arms.idleWrist();
+        }
+    }
```

Comment:
A better pattern might be to have a third `IdleArm` command that tries to keep the arm from moving, and prevents it from extending beyond the frame perimeter. Set the `IdleArm` command as the default command of the `ArmSubsystem`, then have a `ArmTeleop` command that requires the `ArmSubsytem`. Then, when the `ArmTeleop` command finishes, the `IdleArm` is automatically scheduled, ensuring that there is always some code running to make sure the arm doesn't extend past the frame perimeter.